charts/timed/values.yaml

```
@@ -104,6 +104,8 @@ backend:
     admins: []
     # -- Define allowed domains for CORS
     corsAllowedOrigins: []
+    # -- Define name for build projects
+    buildProject: " BUILD"
```

is this whitespace legit?
```suggestion
    buildProject: "BUILD"
```
what's a build project?

---

The space there is on purpose.

Not really sure, but it's part of the Plan-Build-Run Model.
As far as I know we define them by using different cost centers in Timed , e.g. "DEVELOPMENT BUILD"

---

oh, so the commercial "BUILD" and not something technical :sweat_smile: 

Do you know if maybe the existing `.redmine.buildProject` should be merged into this variable? 

https://github.com/adfinis/helm-charts/blob/dd9d46a7907c92f46615e018bdeee1f562410de0/charts/timed/templates/configmap-backend.yaml#L77

---

that one defines the name in Redmine, that is different from the one used in Timed